base_layer/wallet_ffi/src/lib.rs

```diff
@@ -2339,7 +2340,7 @@ pub unsafe extern "C" fn transport_tcp_create(
 pub unsafe extern "C" fn transport_tor_create(
     control_server_address: *const c_char,
     tor_cookie: *const ByteVector,
-    tor_identity: *const ByteVector,
+    _tor_identity: *const ByteVector,
```

Comment:
Probably best to just remove it

Reply:
careful, this will break the Swift and JNI wrappers if we don't update those at the same time.